src/js/SignInUI.js

```diff
@@ -0,0 +1,41 @@
+import AuthAPI from './AuthAPI';
+
+export default class SignIn{
+  user = document.querySelector('#identifier');
+
+  pass = document.querySelector('#login-pass');
+
+  idError = document.querySelector('#identifier-err');
+
+  passError = document.querySelector('#password-err');
+
+  signIn() {
+    const { user, pass, idError, passError, errorLog} = this;
+    let errors = 0;
+    if (user.value.length < 1) {
+      errorLog(`There is an empty Username`, idError);
+      errors++;
+    }if (pass.value.length < 1) {
+      errorLog(`There is an empty Password`, passError);
+      errors++;
```

Comment:
```suggestion
    if (!user.value.trim()) {
      errorLog(`There is an empty Username`, idError);
      errors++;
    }if (!pass.value.trim()) {
      errorLog(`There is an empty Password`, passError);
      errors++;
```

there was no check for spaces `"     "`

Reply:
In addition to this pls fix code formatting